res/css/views/settings/_SetIdServer.scss

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2019 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_SetIdServer .mx_Field_input {
+    width: 300px;
```

Comment:
I'm not sure 300px is right... The input is supposed to be aligned with the other inputs on the page (except add email/phone, which are wrong). 

The common CSS for this is (see `.mx_GeneralUserSettingsTab_languageInput`):
```
margin-right: 100px; // Align with the other fields on the page
```

Reply:
mm - we may have to make all the other fields smaller then, otherwise I don't think we'll have room for the tooltip.

Reply:
done - does this look reasonable?

Reply:
The tooltip will overextend itself because it's the wrong shape anyways.

I'm not entirely sure the mixin is equivalent either. It looks like the math has gone a bit sideways.

Reply:
Empirically it looks correct to me:

<img width="733" alt="Screenshot 2019-08-13 at 16 22 57" src="https://user-images.githubusercontent.com/986903/62954066-a7643200-bde6-11e9-8fe0-74d78d5525b6.png">

Reply:
The inputs are supposed to be ~10px from the left edge of the avatar, and clamped there:

![image](https://user-images.githubusercontent.com/1190097/62954974-83d0c580-bdad-11e9-8d8c-513e358a800f.png)

The inputs are of variable width whereas the "column" the avatar sits in is not.

Reply:
also the section texts, labelled toggles, and that random revoke button all need to line up...

Reply:
mm - ok, that's not the design as it appears in the privacy bit though - there the all the fields in all the sections are the same length and are 31px from the profile picture.

Reply:
it looks like it's using an older version of the settings design. An even still, your screenshot is not 31px :(

Reply:
Oh I see :/ And no, I don't think the tooltip would have fitted with just 31px.